### PR TITLE
(feat) Landing Page Skeleton

### PR DESCRIPTION
**Pull request by misteryeo**
- Refactor and create a new component for Intro as a child of Home
- Create component for customer logos as a child of Home
- Create component for three core features as a child of Home
- Create component for contact/about us as a child of Home

Additional styling will be required next week:
- Looking at implementation of owl carousel for testimonials to be clean
- Should implement a footer and build a favicon
- An element of design for the feature images currently using an old placeholder
- The navbar covers the sections when links are clicked, this has been added to backlog

### CODE REVIEW

File: client/components/homepage/Contact.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            <p>LinkedIn Link</p>
+          </div>
+        </div>
+        <div className="col-md-3">
+          <div className="box-simple">
+            <div className="icon" />
+            <h3>Name</h3>
+            <p>Profile Info Summary</p>
+            <p>Github Link</p>
+            <p>LinkedIn Link</p>
```
Comment (dshelters) [inline client/components/homepage/Contact.jsx]: All of these contact sections seem similar enough that we should probably turn them into a component we can reuse and keep ourselves dry ;)

File: client/components/homepage/Customers.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        </div>
+        <div className="col-sm-4 col-md-2 col-xs-6">
+          <div className="customer"><img src="../img/amazon.png" title="Amazon" alt="" className="img-responsive" /></div>
+        </div>
+        <div className="col-sm-4 col-md-2 col-xs-6">
+          <div className="customer"><img src="../img/hackreactor.png" title="Hack Reactor" alt="" className="img-responsive" /></div>
+        </div>
+        <div className="col-sm-4 col-md-2 col-xs-6">
+          <div className="customer"><img src="../img/hackreactor.png" title="Hack Reactor" alt="" className="img-responsive" /></div>
+        </div>
```
Comment (dshelters) [inline client/components/homepage/Customers.jsx]: Looks like each of these customer sections are pretty similar as well. Probably worth creating a reusable component here as well.

File: client/components/homepage/Testimonials.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            <li className="item">
+              <div className="testimonial same-height-always">
+                <div className="text">
+                  <p>Insert testimonial</p>
+                </div>
+                <div className="bottom">
+                  <div className="testimonial-icon"></div>
+                  <div className="name-picture"><img alt="" src="../img/kay.jpg" />
+                    <h5>Kay Lee</h5>
+                    <p>CEO, PadawanJS</p>
```
Comment (dshelters) [inline client/components/homepage/Testimonials.jsx]: Another opportunity to keep ourselves DRY with these testimonial components

**Comment by misteryeo** [comment]
[Add skeleton and linking for landing page using Bootstrap](https://trello.com/c/NtITwAko/107-add-skeleton-and-linking-for-landing-page-using-bootstrap)

**Pull request merged**